crates/rust-client/src/account/mod.rs

```diff
@@ -77,7 +77,7 @@ use crate::sync::NoteTagRecord;
 pub mod component {
     pub const MIDEN_PACKAGE_EXTENSION: &str = "masp";
 
-    pub use miden_lib::account::auth::*;
+    pub use miden_lib::account::auth::{AuthRpoFalcon512, *};
```

Comment:
If I'm mistaken the specific `use` of `AuthRpoFalcon512` is needless
```suggestion
    pub use miden_lib::account::auth::*;
```